cloudify_kubernetes/workflows.py

```diff
@@ -125,3 +127,75 @@ def update_resource_definition(node_instance_id,
         _params={utils.DEFINITION_ADDITIONS: resource_definition_changes})
     node_instance.logger.info(
         'Executed update in order to push the new changes.')
+
+
+def refresh_and_store_token(ctx,
+                            kubernetes_cluster_node_instance_id,
+                            deployment_capability_name,
+                            service_account_node_instance_id,
+                            secret_token_node_instance_id,
+                            store_token_and_kubeconfig_id):
+
+    cluster_node_instance = get_node_instance_from_node_instance_id_or_node_id(
+        kubernetes_cluster_node_instance_id)
+    execute_node_instance_operation(
+        cluster_node_instance, 'cloudify.interfaces.lifecycle.poststart')
+    execute_node_instance_operation(
+        cluster_node_instance, 'cloudify.interfaces.lifecycle.check_drift')
+
+    create_secrets_kubernetes_config(deployment_capability_name)
+
+    service_account_node_instance = \
+        get_node_instance_from_node_instance_id_or_node_id(
+            service_account_node_instance_id)
+    execute_node_instance_operation(service_account_node_instance,
+                                    'cloudify.interfaces.lifecycle.update')
+    execute_node_instance_operation(service_account_node_instance,
+                                    'cloudify.interfaces.lifecycle.poststart')
+
+    secret_token_node_instance = \
+        get_node_instance_from_node_instance_id_or_node_id(
+            secret_token_node_instance_id)
+    execute_node_instance_operation(secret_token_node_instance,
+                                    'cloudify.interfaces.lifecycle.delete')
+    execute_node_instance_operation(secret_token_node_instance,
+                                    'cloudify.interfaces.lifecycle.creat')
```

Comment:
I would suggest moving all of these to `constants.py` and just import them from there , so it can be reused every where if needed 
```suggestion
                                    'cloudify.interfaces.lifecycle.create')
```